00_download_data/nbconverted/download_data.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
```

Comment:
As mentioned in my comment, I would suggest to wrap the code into a main function, and separate all the data reading sections into methods.

Reply:
Thanks @nrosed! I (and others in the lab) generally export notebooks to Python files to make code review a bit easier, since it's hard to read a diff of the raw JSON in the `.ipynb` files. Since this is a new notebook it doesn't matter as much, but for changes to existing notebooks it can be hard to tell what actually changed and where to make inline comments.

This is a good point, though - someone new to the repo could easily be confused by this. I'll add a note to the README making it clear that this download script is intended to be run as a notebook in a linear fashion.

Reply:
aha, ok makes sense. Thanks for the info!